google/cloud/internal/backoff_policy_test.cc

```diff
@@ -41,6 +41,27 @@ TEST(ExponentialBackoffPolicy, Simple) {
   EXPECT_GE(ms(100), delay);
 }
 
+/// @test Verify the initial and maximum delay are respected.
+TEST(ExponentialBackoffPolicy, RespectMinimumAndMaximumDelay) {
+  ExponentialBackoffPolicy tested(ms(10), ms(12), 2.0);
+
+  auto delay = tested.OnCompletion();
+  EXPECT_LE(ms(10), delay);
```

Comment:
We probably want to `#include "google/cloud/testing_util/chrono_output.h"` here so that test failures are intelligible.